simulation/agents.py

```diff
@@ -468,11 +468,16 @@ def choose_heating_system(
             cost_as_proportion_of_budget = min(
                 costs[heating_system] / self.renovation_budget, multiple_cap
             )
-            weight = 1 / (1 + math.exp(cost_as_proportion_of_budget))
+            weight = 1 / math.exp(cost_as_proportion_of_budget)
```

Comment:
Could also be this, but I think the current way is easier to read
```suggestion
            weight = math.exp(-cost_as_proportion_of_budget)
```